### ISSUE TITLE
Idea: Optionally registering a view for Features

### ISSUE THREAD

**Issue by AidanLaycock** [JustSteveKing/laravel-feature-flags#22]
Hi!

I was wondering what your thoughts are on providing an option to register a view to see the Feature flags/groups etc that are currently implemented across the code base as well as showing the users within the group. As well as showing when a feature should expire through timebombs.

And potentially a longer term feature to add the ability for users to be able to manage who is in feature groups via the UI.

I'm happy to work on this over the next few weeks and to contribute a PR for it, but I'd love to get some feedback to see if it would be useful/if there are any additional bits of functionality that would be useful.

**Comment by JustSteveKing** [comment]
I think this would be a fantastic idea!

**Comment by AidanLaycock** [comment]
@JustSteveKing - Is there anything that you'd want to see in this? Do you also have any preferences over the technology used for this?

I'd like to use the following, but happy to take a steer if you have any preference:

- Blade (Ideally as much as possible so that it can be extended into other stacks easier)
- TailwindCSS (We can bundle this up and know that it shouldn't take much bandwidth)
- Vue.js - Only if needed, and I may try to separate this out so that it's optional.

**Comment by JustSteveKing** [comment]
Yeah I think that would be a good call, maybe something less opinionated than Vue if it can be helped - but I understand it is sometimes needed.

**Comment by AidanLaycock** [comment]
Great, I'll try to not use JS where possible so that it will integrate better with existing apps.

I'll start working on this over the next couple of weeks and I'll send some screenshots/WIP when I have time as well.
